stable/vsphere-cpi/templates/daemonset.yaml

```diff
@@ -22,37 +25,39 @@ spec:
     type: RollingUpdate
   template:
     metadata:
-{{- if .Values.daemonset.podAnnotations }}
+      {{- if .Values.daemonset.podAnnotations }}
       annotations:
-{{ toYaml .Values.daemonset.podAnnotations | indent 8 }}
-{{- end }}
+      {{- toYaml .Values.daemonset.podAnnotations | indent 8 }}
+      {{- end }}
       labels:
         app: {{ template "cpi.name" . }}
-        component: cloud-controller
+        component: cloud-controller-manager
+        tier: control-plane
         release: {{ .Release.Name }}
         vsphere-cpi-infra: daemonset
-{{- if .Values.daemonset.podLabels }}
-{{ toYaml .Values.daemonset.podLabels | indent 8 }}
-{{- end }}
+        {{- if .Values.daemonset.podLabels }}
+        {{- toYaml .Values.daemonset.podLabels | indent 8 }}
+        {{- end }}
     spec:
       nodeSelector:
-{{- if .Values.daemonset.nodeSelector }}
-{{ toYaml .Values.daemonset.nodeSelector | indent 8 }}
-{{- else }}
+      {{- if .Values.daemonset.nodeSelector }}
+      {{- toYaml .Values.daemonset.nodeSelector | indent 8 }}
+      {{- else }}
         node-role.kubernetes.io/master: ""
-{{- end }}
-      securityContext:
-        runAsUser: 0
+      {{- end }}
       tolerations:
-{{- if .Values.daemonset.tolerations }}
-{{ toYaml .Values.daemonset.tolerations | indent 8 }}
-{{- else }}
+      {{- if .Values.daemonset.tolerations }}
+      {{- toYaml .Values.daemonset.tolerations | indent 8 }}
+      {{- else }}
       - key: node.cloudprovider.kubernetes.io/uninitialized
```

Comment:
I feel these tolerations should always be added, regardless of additional tolerations specified by the user.

Reply:
Ok... if they should always be added, that's fine. I was thinking of some of these other orchestrators that don't use the same labels